src/Button/index.jsx

```diff
@@ -0,0 +1,225 @@
+/*
+ * Copyright (c) 2017-2018 dunnhumby Germany GmbH.
+ * All rights reserved.
+ *
+ * This source code is licensed under the MIT license found in the LICENSE file
+ * in the root directory of this source tree.
+ *
+ */
+
+import React                          from 'react';
+import PropTypes                      from 'prop-types';
+
+import { generateId } from '../utils';
```

Comment:
Great work! Just a minor lint tweak here and it's good to go 🚀

Reply:
@daniel-martic-sociomantic Sure! ;)